modules/backend/models/BrandSetting.php

```diff
@@ -115,6 +116,17 @@ public static function getLogo()
         return self::getDefaultLogo() ?: null;
     }
 
+    public static function getFallbackImage()
+    {
+        $settings = self::instance();
+
+        if ($settings->fallback_image) {
+            return $settings->fallback_image->getDiskPath();
```

Comment:
This is too tightly coupled to the local storage driver, especially in coordination with the storage_path('app' . $filePath) from the related PR. Additionally, this should be able to be configured with `brand.fallbackImagePath` just like the logo and favicon can

Reply:
Making it configurable with brand.fallbackImagePath is a matter of including a getDefaultFallbackImage() method in BrandSettings.php similar to those of the logo and favicon. I can add this as a commit. But regarding the removal of the storage_path, I’m uncertain what can be done. What do you suggest?

Reply:
You should be returning the File object, not a path, and then calling methods on the File object to get its content through the flysystem abstraction